### PR TITLE
Issues 3196 3197 links

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@callahad or @6a68 - could you review this one?

Two changes that @skinny97214 asked me to make - Add an "Account" link to the nav bar if the user is signed in and move "Develoers" to the footer.

### CODE REVIEW

**Comment by callahad** [comment]
This looks pretty reasonable to me.

File: resources/views/layout.ejs [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
         <ul class="nav cf">
             <li><a href="/about"><%= gettext("How it works") %></a></li>
-            <li><a href="https://developer.mozilla.org/docs/persona" target="_blank"><%= gettext("Developers") %></a></li>
+            <li class="signOut"><a href="/"><%= gettext("Account") %></a></li>
```
Comment (callahad) [inline resources/views/layout.ejs]: Why the `signOut` class here?

Comment (shane-tomlinson) [inline resources/views/layout.ejs]: It's a mis-named class that means "show when the user is signed in". I was not sure whether I should expand the scope of the PR to rename it or not. I'd much rather re-name it if you agree.

**Comment by callahad** [comment]
Shane is going to add a commit to rename the class, then this will be good to go.

**Comment by shane-tomlinson** [comment]
@callahad  - could you re-review this?

**Comment by callahad** [comment]
Looks trivially correct. Pulling and re-checking now...

**Pull request merged**